htdocs/projet/tasks/contact.php

```diff
@@ -270,204 +276,13 @@
 
 		dol_fiche_end();
 
-		/*
-		 * Lignes de contacts
-		 */
-		print '<br>';
-/*
 		// Contacts lines (modules that overwrite templates must declare this into descriptor)
 		$dirtpls=array_merge($conf->modules_parts['tpl'],array('/core/tpl'));
 		foreach($dirtpls as $reldir)
 		{
-		    $res=@include dol_buildpath($reldir.'/contacts.tpl.php');
-		    if ($res) break;
+			$res=@include dol_buildpath($reldir.'/contacts.tpl.php');
```

Comment:
Mmm, all tabs is OK for me. But the syntactic coloration let me know that there was mixed indentation in the same line.
But don't sweat it, absolutely tiny issue.
I don't mind fixing it myself at all.
The loadBoard() would be really nice though.